docs/README.md

```diff
@@ -868,6 +872,274 @@ In this setup there is a Primary cluster (`cluster1`) and a Remote cluster (`clu
     kubectl delete ns sample --context="${CTX_CLUSTER2}"
     ```
 
+### External-Controlplane
+
+These instructions install an [external-controlplane](https://istio.io/latest/docs/setup/install/external-controlplane/) Istio deployment using the Sail Operator and Sail CRDs. **Before you begin**, ensure you complete the [common setup](#common-setup).
+
+These installation instructions are adapted from: https://istio.io/latest/docs/setup/install/external-controlplane/ and are intended to be run in a development environment such as `kind` and not in production.
+
+In this setup there is an External Controlplane cluster (`cluster1`) and a Remote cluster (`cluster2`) which are on separate networks.
+
+1. Create an `Istio` resource on `cluster1` to manage the ingress gateways for the External Controlplanes.
+
+    ```sh
+    kubectl create namespace istio-system --context "${CTX_CLUSTER1}"
+    kubectl apply --context "${CTX_CLUSTER1}" -f - <<EOF
+    apiVersion: sailoperator.io/v1alpha1
+    kind: Istio
+    metadata:
+      name: default
+    spec:
+      version: v${ISTIO_VERSION}
+      namespace: istio-system
+      values:
+        pilot:
+          resources:
+            requests:
+              cpu: 100m
+              memory: 1024Mi
```

Comment:
I'm thinking we should introduce a new profile called `kind` instead of repeating this in all the examples. Better yet, check if this is really (still?) needed when deploying kind with defaults and if not, simplify remove this.

Reply:
I don't think it's needed. Maybe in CI where resources are more constrained? But IMO we don't need this in the examples. I probably just copy/pasta'd or branched off an old example that included this. I will remove it.